beacon_node/execution_layer/src/engine_api.rs

```diff
@@ -511,6 +554,7 @@ impl<E: EthSpec> ExecutionPayloadBodyV1<E> {
                     ))
                 }
             }
+            ExecutionPayloadHeader::Eip6110(_) => todo!(),
```

Comment:
I currently can't wrap my head around this. How do I get the correct value for `deposit_receipts` here? Do I have to add `deposit_receipts` to `engine_api::ExecutionPayloadBodyV1<E>`?

Reply:
Yeah, eventually you should probably add the `deposit_receipts` field to the `ExecutionPayloadBodyV1` type. You'll also need the EL to add support for it to their [`engine_getPayloadBodiesByRangeV1`](https://github.com/ethereum/execution-apis/blob/main/src/engine/shanghai.md#engine_getpayloadbodiesbyrangev1) call.

As a workaround for initial testing you could use `--prune-payloads false`. Then this code-path won't even be used. It's only used in the case where Lighthouse has deleted the payload body and re-fetches it from the EL (usually for a peer, or the HTTP API).